src/App.js

```diff
@@ -29,7 +51,14 @@ class App extends Component {
                         </Navbar.Header>
                         <Navbar.Collapse>
                             <Nav pullRight>
-                                <NavDropdown eventKey={1} id="basic-nav-dropdown" noCaret title="Learn">
+                                <NavDropdown
+                                    eventKey={1}
+                                    title="Learn"
+                                    id="basic-nav"
+                                    onMouseEnter={this.handleLearnMenuItemOpen}
+                                    onMouseLeave={this.handleMenuItemClose}
+                                    open={this.state.isLearnMenuItemOpen}
```

Comment:
noticed a console error, we need to pass `onToggle` handler if we want to override `open`.
https://5c507d49471426000887a6a7--react-bootstrap.netlify.com/components/dropdowns/#btn-dropdowns-props-dropdown
you'll need to do something like this for all dropdowns where you're passing in `open` value.
```suggestion
                                    onToggle={isOpen =>
                                        isOpen ? this.handleLearnMenuItemOpen : this.handleMenuItemClose
                                    }
                                    open={this.state.isLearnMenuItemOpen}
```